ouroboros-consensus/src/Ouroboros/Consensus/Ledger/Query.hs

```diff
@@ -285,6 +285,8 @@ class QuerySat (mk :: MapKind) (fp :: FootprintL)
 
 instance QuerySat mk SmallL
 
+instance QuerySat EmptyMK LargeL
```

Comment:
I'm not sure about this. How can empty tables satisfy large query?

Reply:
Added a comment explaining it